dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/CollectionRepositoryImpl.java

```diff
@@ -292,6 +297,68 @@ protected String getColumnName(String modelProperty) {
     }
   }
 
+  @Override
+  public SearchPageResponse<DigitalObject> getDigitalObjects(
+      UUID collectionUuid, SearchPageRequest searchPageRequest) {
+    final String doTableAlias = digitalObjectRepositoryImpl.getTableAlias();
+    final String doTableName = digitalObjectRepositoryImpl.getTableName();
+
+    String commonSql =
+        " FROM "
+            + doTableName
+            + " AS "
+            + doTableAlias
+            + " LEFT JOIN collection_digitalobjects AS cd ON "
+            + doTableAlias
+            + ".uuid = cd.digitalobject_uuid";
+
+    Map<String, Object> argumentMappings = new HashMap<>();
+    argumentMappings.put("uuid", collectionUuid);
+
+    if (StringUtils.hasText(searchPageRequest.getQuery())) {
```

Comment:
Könnte man `searchPageRequest.getQuery()` in eine Variable auslagern? Wird ja weiter unten nochmal aufgerufen.

Reply:
ist geändert